apps/src/code-studio/pd/workshop_dashboard/components/survey_results/survey_rollup_table.jsx

```diff
@@ -95,6 +105,12 @@ export class SurveyRollupTable extends React.Component {
     }));
   }
 
+  renderAverage(value, category) {
+    return value
+      ? `${value.toFixed(2)} / ${this.categoryDenominator[category]}`
```

Comment:
@hacodeorg this actually seems like a good opportunity to change our rounding to the 10ths place, from the 100ths. This wasn't specced but something i intended to eventually do after my site visit

Reply:
Will do